numpy/core/src/multiarray/methods.c

```diff
@@ -1179,7 +1179,7 @@ array_resize(PyArrayObject *self, PyObject *args, PyObject *kwds)
         return NULL;
     }
 
-    ret = PyArray_Resize(self, &newshape, refcheck, NPY_CORDER);
+    ret = PyArray_Resize(self, &newshape, refcheck, NPY_ANYORDER);
```

Comment:
The order is ignored (that is documented even), ignoring the order is pretty much the same as `NPY_ANYORDER`, so that seems clearer.